source/adapters/native_cpu/queue.cpp

```diff
@@ -83,5 +83,6 @@ UR_APIEXPORT ur_result_t UR_APICALL urQueueFinish(ur_queue_handle_t hQueue) {
 UR_APIEXPORT ur_result_t UR_APICALL urQueueFlush(ur_queue_handle_t hQueue) {
   std::ignore = hQueue;
 
-  DIE_NO_IMPLEMENTATION;
+  // The native cpu queue is greedy, flushes are a no-op.
+  return UR_RESULT_SUCCESS;
```

Comment:
Is this a behavior change?

Reply:
yeah it was breaking a fixture, not sure how correct this is so I'll revert and just KNOWN_FAILURE_ON the effected tests with a comment